spell/spell-reviewer-goerli-checklist.md

```diff
@@ -2,10 +2,6 @@
 
 ## Goerli
 
-PR: https://github.com/makerdao/spells-goerli/pull/TODO
-
-Exec date: YYYY-MM-DD
```

Comment:
If we are removing this let's make sure that there is a check somewhere else for the date being correct, as this functioned as a defacto check

Reply:
done in 67a622e, to check the spell `description` date when generating the exec hash

Reply:
included in the crafter checklist as well in 788043c